### PR TITLE
Added a media queries for viewport until 320px

### PR DESCRIPTION
**Pull request by JaabaShooler**
Table on the search queries page is now adaptive for viewport until 320px
image

### CODE REVIEW

File: src/components/Tools/Table/Table.tsx [hunk 1]
```diff
@@ -32,14 +32,33 @@ const Table: React.FC<ITableProps> = ({sortBy, sortOrder, data, header, prevSort
...
-                <label className="Table__element">{index + 1}.</label>
-                <label className="Table__element">{query.firstColumn}</label>
-                <label className="Table__element Table__element--right">{query.secondColumn}</label>
-                <label className="Table__element Table__element--right">{query.thirdColumn}</label>
-                <label className="Table__element Table__element--right">{query.fourthColumn}</label>
+                <label data-label={'#'} className="Table__element">
+                  {index + 1}.
+                </label>
+                <label data-label={header[0].title} className="Table__element">
+                  {query.firstColumn}
```
Comment (hmelenok) [inline src/components/Tools/Table/Table.tsx:42]: why just not create a map of `<label />` elements? what about the fifth element? what about sixth? What about n element?

**Review by hmelenok — CHANGES_REQUESTED** [review]
Fix table cols rendering, to not rely on property name

File: src/components/Tools/Table/Table.tsx (hunk 1 above)
Comment (JaabaShooler) [reply src/components/Tools/Table/Table.tsx]: Ok, its done i think

File: src/components/Tools/Table/Table.scss [hunk 2]
```diff
@@ -113,3 +120,55 @@
...
+      justify-content: space-between;
+      &::before {
+        content: attr(data-label);
+        font-weight: bold;
+        min-width: 50px;
+        text-align: left;
+      }
+    }
+  }
+}
```
[the remark this answers is not in the document]
Comment (JaabaShooler) [reply src/components/Tools/Table/Table.scss]: my bad

**Review by hmelenok — APPROVED** [review]

**Pull request merged**